tests/strategies/filter/test_crop_filter.py

```diff
@@ -0,0 +1,42 @@
+"""Tests the crop filter strategy."""
+import os
+from pathlib import Path
+
+
+def test_crop_filter():
+    """Test the crop filter strategy on 'sample_1280_853.jpeg',
+    downloaded from filesamples.com (called 'sample_1280x853').
+
+    Note: This test incorporates much of the contents of the test
+    'test_jpeg.py', so if that test fails, this one should fail too.
+    """
+    from oteapi.models.filterconfig import FilterConfig
+    from oteapi.models.resourceconfig import ResourceConfig
+    from oteapi.strategies.filter.crop_filter import CropFilter
+    from oteapi.strategies.parse.image import ImageDataParseStrategy
+
+    parent_dir = Path(__file__).resolve().parents[1]
```

Comment:
```suggestion
    parent_dir = Path(__file__).resolve().parent
```

Reply:
No, `parents[1]` is the parent folder of `parent` (i.e. `parent.parent`).

Just `parent` corresponds to `parents[0]`.